stable/datadog/templates/daemonset.yaml

```diff
@@ -274,6 +284,16 @@ spec:
             path: {{ default "/var/lib/datadog-agent/logs" .Values.datadog.logsPointerHostPath | quote }}
           name: pointerdir
         {{- end }}
+        {{- if (and (.Values.datadog.logsEnabled) (.Values.datadog.logsConfigContainerCollectAll)) }}
+        - hostPath:
+            path: /var/log/pods
+          name: logpodpath
+        {{- end }}
```

Comment:
I think this `{{- end }}` should be move after line 296, in order to not add "logcontainerpath" hostPath if logs are not activated